unit-tests/source_tree_test.py

```diff
@@ -12,6 +12,27 @@
 from fab.source_tree import ExtensionVisitor, TreeDescent, TreeVisitor
 from fab.queue import QueueManager
 
+from multiprocessing import Manager
+
+
+tracker = Manager().list()
```

Comment:
Could this be a static class variable? i.e. defined at the top level of the class:

```python
class Whatever:
    static_var = 'foo'
```

Reply:
I thought I tried something like this and it didn't like it but I will have another go

Reply:
This was rendered moot by the change lower down (since the unit tests no longer care about the queue)